src/cluster_legacy.c

```diff
@@ -5797,6 +5797,10 @@ void clusterUpdateSlots(client *c, unsigned char *slots, int del) {
     }
 }
 
+/* Get the replication offset of a node.
+ *
+ * Nodes do not update their own information in the cluster node list,
+ * so for myself node, we cannot use node->repl_offset directly. */
```

Comment:
This isn't just related to cluster nodes, right?

```suggestion
 * Nodes do not update their own information in the cluster state,
 * so for self node, we cannot use `node->repl_offset` directly. */
```